services/sidecar/docker/boot.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+#
+
+# BOOTING application ---------------------------------------------
+echo "Booting in ${MY_BOOT_MODE} mode ..."
+
+
+if [[ ${MY_BUILD_TARGET} == "development" ]]
+then
+  echo "  User    :`id $(whoami)`"
+  echo "  Workdir :`pwd`"
+  echo "  Environment :"
+  printenv  | sed 's/=/: /' | sed 's/^/    /' | sort
+  #--------------------
+
+  cd services/sidecar
+  $MY_PIP install --user -r requirements/dev.txt
+  cd /devel
+
+  DEBUG_LEVEL=debug
+  #--------------------
+  echo "  Python :"
+  python --version | sed 's/^/    /'
+  which python | sed 's/^/    /'
+  echo "  PIP :"
+  $MY_PIP list | sed 's/^/    /'
+
+
+elif [[ ${MY_BUILD_TARGET} == "production" ]]
+then
+  DEBUG_LEVEL=info
+fi
+
+
+# RUNNING application ----------------------------------------
+if [[ ${BOOT_MODE} == "debug" ]]
+then
+  # TODO: activate pdb??
+  DEBUG_LEVEL=debug
+  CONCURRENCY=1
+else
+  CONCURRENCY=2
```

Comment:
We should keep an eye on that. Concurrency should be done via scaling the sidecars. I just added the two for debugging

Reply:
I kept this as it was. I guess this means that every sidecar has actually TWO workers which is kind of wrong, provided that they share the same input/output folder!!! PLEASE comment on this.